util/src/test/java/io/zeebe/util/ObjectWriterFactoryTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.util;
+
+import static io.zeebe.util.ObjectWriterFactory.getDefaultJsonObjectWriter;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+import org.springframework.util.unit.DataSize;
+
+public class ObjectWriterFactoryTest {
+
+  @Test
+  public void shouldReturnNonNullDefaultObjectWriter() {
+    // when
+    final ObjectWriter actual = getDefaultJsonObjectWriter();
+
+    // then
+    assertThat(actual).isNotNull();
+  }
+
+  @Test
+  public void shouldReturnObjectWriterWithPrettyPrintingEnabled() {
+    // given
+    final Map<String, String> objectToSerialize = new HashMap<>();
+    objectToSerialize.put("field1", "value1");
+    objectToSerialize.put("field2", "value2");
+
+    String actual;
+    // when
+    try {
+      actual = getDefaultJsonObjectWriter().writeValueAsString(objectToSerialize);
+    } catch (JsonProcessingException e) {
+      fail(e.getMessage());
+      actual = "";
+    }
+
+    // then
+    assertThat(actual)
+        .withFailMessage("Should be pretty printed and therefore contain line breaks")
+        .contains(System.lineSeparator());
+  }
+
+  @Test
+  public void shouldReturnObjectWriterThatWritesDurationAsNumberInSeconds() {
+    // given
+    final Duration objectToSerialize = Duration.ofSeconds(30);
+
+    String actual;
+    // when
+    try {
+      actual = getDefaultJsonObjectWriter().writeValueAsString(objectToSerialize);
+    } catch (JsonProcessingException e) {
+      fail(e.getMessage());
+      actual = "";
+    }
+
+    // then
+    assertThat(actual).describedAs("Serialized form of Duration").isEqualTo("30.000000000");
```

Comment:
Wouldn't it be better if duration is serialized to readable format like "30s"?

Reply:
To be honest, I am not sure. I did think about it. And I kinda weighed against it. I was very happy to find a standard implementation to serialize Durations and also other date and time formats. So I decided to use this standard implementation.

If I would write my own serializer, I would probably output in in ISO format (something like "PT30S"). I wouldn't output it as 30s, because this is very custom. 

One insight I got from talking to Alistair of the cloud team is that hum readable has disadvantages when you want to parse such a configuration, or write such a configuration with a machine. In effect, they want to generate configurations through code. Now, I know this doesn't apply to this location very well. However, I try to strike a blance between human readable and machine readable/writable.

Having said all that: I can change it, if you want to. If it's just a mild preference on your part, I would probably leave it for a time being.

Reply:
It is a mild preference from my side. As far as I understand this is only indented for humans to read. It is difficult for me to guess the timeout is in seconds by just reading the serialized config. We also have some other timeout config which are not duration but just long values representing milliseconds (we should probably make them consistent, out of scope for this pr). So it could be confusing. 

I saw that `JavaTimeModule` has a config to serialize duration into ISO format.

Reply:
Good find. Changed the setting and it is now output as ISO format.

Reply:
If you want, you can look at the changes. Otherwise, I would merge it this evening.

Reply:
Thanks :+1: